pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -68,7 +68,6 @@ stdenv.mkDerivation rec {
     cd "$out/etc/fonts"
     rm conf.d/{50-user,51-local}.conf
```

Comment:
@Balletie I moved `30-nixos-generic-alias.conf` to the `local.conf` 51 priority, it work almost perfect beside that `fontconfig-ultimate`'s [`43-wqy-zenhei-sharp.conf`](https://github.com/bohoomil/fontconfig-ultimate/blob/master/fontconfig_patches/fonts-settings/43-wqy-zenhei-sharp.conf) set deja-vu and WenQuanYi as preferred fonts, taking priority over rules defined in `local.conf`.

I submitted an upstream [issue](https://github.com/bohoomil/fontconfig-ultimate/issues/173), and propose that in the meantime we patch fontconfig-ultimate to move `43-wqy-zenhei-sharp.conf` to `60-wqy-zenhei-sharp.conf` as it is a generic->family configuration.

Do you think this is an acceptable solution?